pkg/virt-operator/resource/apply/rbac.go

```diff
@@ -34,49 +34,50 @@ func (r *Reconciler) createOrUpdateRoleBinding(rb *rbacv1.RoleBinding, imageTag
 	return r.createOrUpdate(rb, imageTag, imageRegistry, id, true)
 }
 
-func (r *Reconciler) createOrUpdate(role runtime.Object,
+func (r *Reconciler) createOrUpdate(required runtime.Object,
```

Comment:
This function name needs to be changed. as its part of `Reconciler` it reads as if any object can be passed to it and will be created or updated.

Reply:
There are 4 dedicated functions to reconcile rbac objects which are `createOrUpdateClusterRole`, `createOrUpdateClusterRoleBinding`, `createOrUpdateRole` and `createOrUpdateRoleBinding`. This is meant to serve as a private helper function.

Thanks, you're right, it is confusing.
I think that I will change this function from being a "method" into being a regular private function to reduce the confusion. Will also rename the function to something like `rbacObjectUpdateHelper` so that its purpose will be clear.

What do you think?